packages/components/react/src/Nav.tsx

```diff
@@ -109,14 +109,12 @@ function renderParts(navList: NavList, currentLesson: Lesson) {
                     navStyles.AccordionTrigger,
                     'flex items-center gap-1 w-full hover:text-primary-700',
                     {
-                      'font-semibold': isPartActive,
+                      [`font-semibold ${navStyles.AccordionTriggerActive}`]: isPartActive,
                     },
                   )}
                 >
-                  <span
-                    className={`${navStyles.AccordionTriggerIcon} i-ph-caret-right-bold scale-80 text-tk-elements-breadcrumbs-dropdown-accordionToggleIconColor`}
-                  ></span>
-                  <span className={navStyles.AccordionTriggerText}>{`Part ${partIndex + 1}: ${part.title}`}</span>
+                  <span className={`${navStyles.AccordionTriggerIcon} i-ph-caret-right-bold scale-80`}></span>
+                  <span>{`Part ${partIndex + 1}: ${part.title}`}</span>
```

Comment:
Oh I hadn't realised that we injected `Part <index>` in the title. This is not ideal when writing a Tutorial in a language other than English.

I feel like we should remove it but not in this PR though.

Curious to hear your thoughts?

Reply:
OH you're right! We should do this differently.